src/confd/yang/infix-services.yang

```diff
@@ -104,6 +124,48 @@ module infix-services {
       }
     }
   }
+  container ssh {
+    description "Configuration for the SSH daemon";
+    leaf enabled {
+      description "Disable or enable SSH daemon";
```

Comment:
Swapping disable/enable here would be more in line with the rest of our service descriptions.